sporkfish/endgame_tablebases/endgame_tablebase_config.py

```diff
@@ -0,0 +1,21 @@
+from typing import Optional
+
+from sporkfish.configurable import Configurable
+
+
+class EndgameTablebaseConfig(Configurable):
+    """Configuration class for an endgame tablebase."""
+
+    def __init__(
+        self,
+        endgame_tablebase_path: Optional[str] = None,
+        query_db: Optional[str] = None,
```

Comment:
Could this be an enum instead, also not sure if it should default to None, looks like currently even if it's None it will query local - doesn't seem intended